source/core/TypeAndDataReflection.cpp

```diff
@@ -27,13 +27,13 @@ namespace Vireo
 
 class DataReflectionVisitor : public TypeVisitor
 {
-public:
```

Comment:
Google style guides and cpplint say private, protected, and public: labels should be indented by exactly one space, even though the recommended indentation for other code is 4 spaces.